app/assets/stylesheets/application.scss

```diff
@@ -12,4 +12,41 @@
  *
  *= require_tree .
  *= require_self
+ *= require foundation_and_overrides
  */
+
+.top-bar {
+  margin-bottom: 30px;
+}
+
+.name {
+  display: inline;
+}
+
+.book {
+  height: 240px;
+  padding: 0;
+  text-align: center;
+  h4 {
+    margin-top: 2em;
+  }
+}
+
+.dashed {
+  border: 2px dashed #808080;
```

Comment:
Color literals like `#808080` should only be used in variable declarations; they should be referred to via variable everywhere else.